addon/components/mdc-checkbox.js

```diff
@@ -38,7 +40,7 @@ export default Component.extend(MDCComponent, SupportsBubblesFalse, {
    * @type {Function}
    * @param {Boolean} checked
    */
-  onchange: x => x,
+  onchange: (x) => x,
```

Comment:
I allowed them to change to the new defaults. Please confirm you want non-default settings.

Reply:
If they are supported as options, yes. I'd favor that over changing big chunks of the codebase as part of this dependency update PR.

Reply:
There is an option for this, but my personal preference is to always use prettier defaults rather than bikeshed. Big chunks of this codebase are being changed anyway.